api/generator/generator.go

```diff
@@ -266,22 +266,20 @@ func (cg *CertGeneratorHandler) Handle(w http.ResponseWriter, r *http.Request) e
 			"certificate":         certSum,
 		},
 	}
+
+	if len(req.Request.Hosts) == 0 {
+		return api.SendResponseWithMessage(w, result,
+			"This certificate lacks a \"hosts\" field. This makes it unsuitable for websites. " +
+			"For more information see the Baseline Requirements for the Issuance and Management of " +
+			"Publicly-Trusted Certificates, v.1.1.6, from the CA/Browser Forum (https://cabforum.org). " +
+			"Specifically, section 10.2.3 (\"Information Requirements\").",
+			errors.New(errors.PolicyError, errors.InvalidRequest).ErrorCode)
+	}
+
 	return api.SendResponse(w, result)
 }
 
-// CSRValidate contains the default validation logic for certificate requests to
-// the API server. This follows the Baseline Requirements for the Issuance and
-// Management of Publicly-Trusted Certificates, v.1.1.6, from the CA/Browser
-// Forum (https://cabforum.org). Specifically, section 10.2.3 ("Information
-// Requirements"), states:
-//
-// "Applicant information MUST include, but not be limited to, at least one
-// Fully-Qualified Domain Name or IP address to be included in the Certificate’s
-// SubjectAltName extension."
+// CSRValidate always returns true
```

Comment:
On the surface, this comment should be a complete sentence.

Ideally, if it always returns true, it's probably not useful to have; its existence should probably be reviewed.